agent/exec/dockerapi/container.go

```diff
@@ -479,6 +480,7 @@ func (c *containerConfig) resources() enginecontainer.Resources {
 	return resources
 }
 
+//nolint:unused // TODO(thaJeztah) this is currently unused: is it safe to remove?
 func (c *containerConfig) virtualIP(networkID string) string {
```

Comment:
I initially removed things like these (as they are unused), but decided to keep that for a later discussion